tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -390,7 +390,12 @@ def shifted_QR(Vm, Hm, fm, evals, k, p, which):
     krylov_vectors = jax.ops.index_update(krylov_vectors, jax.ops.index[0:k, :],
                                           Vk)
     krylov_vectors = jax.ops.index_update(krylov_vectors, jax.ops.index[k:], v)
-    return krylov_vectors, H, fk
+    Z = jax.numpy.linalg.norm(fk)
+    #if fk is a zero-vector then arnoldi has exactly converged.
+    #use small threshold to check this
+    converged = jax.lax.cond(Z < res_thresh, lambda x: True, lambda x: False,
+                             None)
```

Comment:
Can't you just do `converged = Z < res_thresh`?

Reply:
lol, you're right ...